### ISSUE TITLE
Feature Request - Release 55.0.0

### ISSUE THREAD

**Issue by amaitland** [cefsharp/CefSharp#1893]
For those interested in sponsoring a `55.0.0` release then please check [Bounty Source](https://www.bountysource.com/issues/40201096-feature-request-release-55-0-0)

There are a few conditions that need to be met before an official `55.0.0` release can happen.

- The following `CEF` issue needs to be resolved https://bitbucket.org/chromiumembedded/cef/issues/1995/migrate-from-breakpad-to-crashpad
- A stable `CEF` build needs to appear on http://opensource.spotify.com/cefbuilds/index.html (currently the only `2883` builds are based on `Beta` releases of `Chromium`.

If the set `BountySource` target is not met then all is not lost, you can of course build your own `Nuget` packages and host then on your local feed. If another contributor to the project wishes to take charge of the release then by all means say so and I'll happily close this issue.

Constructive comments welcome.

**Comment by smaranh** [comment]
you might also want to add it to the Readme.md.

**Comment by GrabzIt** [comment]
@smaranh great idea!! I have added a PR to try to encourage people to contribute.

**Comment by amaitland** [comment]
Firstly thank you for the amazing show of support! As a gesture here is a link to the `General Usage Guide` I had been working on, it started as a port of https://bitbucket.org/chromiumembedded/cef/wiki/GeneralUsage and now has many `CefSharp` specific details.

https://gist.github.com/amaitland/aa976285407ba35cf895c67c614ba82c

Please review at your leisure, if you have any requests for topics you'd like to see covered then let me know. This `Guide` will be included as part of the release and will replace large parts of the existing `FAQ` (The `Javascript` specific parts at a minimum).

If/when `50%` of the target is reached I will release a new `-pre` set of packages to `Nuget`. The `crashpad` issue listed above would require you to add a small block of code to your application to deal with launching of the new `crashpad` process type. It's relatively trivial, and details are in https://github.com/cefsharp/CefSharp/issues/1876#issuecomment-268422985 Is everyone happy to see a `-pre` release before this issue is resolved? There will be a application crash if the additional code isn't added. Feedback welcome 👍

**Comment by amaitland** [comment]
If/when `75%` of the target is met I will attempt to add `GitLink` support as outlined in #1680 

In theory `GitLink` will allow you to step through the `CefSharp` code hosted on `GitHub` by rewriting the `pdb` source file locations.

**Comment by jankurianski** [comment]
My company appreciates all the effort 👍

**Comment by joe-williams-cccu** [comment]
@jankurianski Thank you and your colleagues!

**Comment by amaitland** [comment]
`GitLink` support has been added, so you can now step into the code when developing. See #1680 for full details.

**Comment by amaitland** [comment]
`Crashpad` support has been resolved `upstream`, hopefully the changes will appear in the `2883` branch shortly. 

https://bitbucket.org/chromiumembedded/cef/issues/1995/migrate-from-breakpad-to-crashpad#comment-33539720

**Comment by amaitland** [comment]
`master` has been upgraded to `3.2883.1542` which is based on `Chromium 55.0.2883.87` (The current stable release). Just waiting on the `crashpad` changes to be merged into `2883` before a `-pre` set of packages is available.

**Comment by amaitland** [comment]
Upgraded to `CEF 3.2883.1545` for which `crashpad` is now disabled by default.

- New [cefsharp/55](https://github.com/cefsharp/CefSharp/tree/cefsharp/55) branch has been created
- I've tagged [55.0.0-pre01](https://github.com/cefsharp/CefSharp/releases/tag/v55.0.0-pre01) and the packages should appear on `MyGet` shortly. They still require testing before uploading to `Nuget.org`

**Comment by amaitland** [comment]
The `55.0.0-pre01` packages have been pushed to `Nuget.org`, should appear in the next few hours (they're awaiting indexing).

Please test them out and report back. If you do encouner a bug then please make sure you fill out the [Bug Report](https://github.com/cefsharp/CefSharp/blob/master/ISSUE_TEMPLATE.md#bug-report) when opening an issue.

**Comment by amaitland** [comment]
Does anyone have feedback? I will look to update to the latest CEF build and release the final version in about a week.

**Comment by philipbeber** [comment]
I tried 55.0.0-pre01 with our app and it worked great! Looking forward to the final release :)

**Comment by amaitland** [comment]
Just as an update.

When a build of https://bitbucket.org/chromiumembedded/cef/commits/f984155b3f0ad80833742d17cdab2065f3ec75ce?at=2883 appears on http://opensource.spotify.com/cefbuilds/index.html.

Should be in the next day or two.

**Comment by amaitland** [comment]
Going with `3.2883.1552` instead as the two newer commits are for `MacOS` and fixing a bug introduced by the `MacOS` change.

`cefsharp/55` and `master` have been updated, see https://github.com/cefsharp/CefSharp/commit/2ded106d59250af124026c8b0d725b717a9123b5 for the relevant details.

**Comment by danielearwicker** [comment]
I'm in the early days of trying this out. I have to say on what I've seen so far, this project is nothing short of a miracle. Thank you!

Also 55.0.0-pre01 seems fine and fixes https://github.com/cefsharp/CefSharp/issues/1819. 😎

**Comment by amaitland** [comment]
Packages have been generated, will confirm when they're pushed to `Nuget.org`.

https://ci.appveyor.com/project/cefsharp/cefsharp/build/55.0.0-CI2139

**Comment by amaitland** [comment]
Packages have been pushed to `Nuget.org`, they should appear shortly (takes a while before they're indexed)

Check out the [Release Notes](https://github.com/cefsharp/CefSharp/releases/tag/v55.0.0) which as always includes a list of **Known Issues**. If you find a bug then please fill out the [Issue Template](https://github.com/cefsharp/CefSharp/blob/master/ISSUE_TEMPLATE.md#bug-report) when opening an issue.

I'll leave this open for a week to let the dust settle.

**Comment by amaitland** [comment]
Generated [API Doc](http://cefsharp.github.io/api/55.0.0/) is available.
